blazor/rating/selection.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Selection in Blazor Rating Component | Syncfusion
+description: Checkout and learn here all about Selection with Syncfusion Blazor Rating component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: Rating
+documentation: ug
+---
+
+# Selection in Blazor Rating Component
+
+All the items before the selected item should be in selected state in a rating component.
+
+Below example demonstrates Selection of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating Value="3"></SfRating>
+
+```
+
+![Blazor Rating Component with Selction](./images/blazor-rating-selection.png)
+
+## Selected Value
+You can get the current updated rating item value selected by the user by setting [Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_Value) property.
```

Comment:
"You can get the currently rated value selected by using the [Value]() property."

Reply:
I have updated it as per comments.